charts/spire/charts/tornjak-frontend/templates/deployment.yaml

```diff
@@ -57,7 +61,7 @@ spec:
             - name: env
               mountPath: {{ include "tornjak-frontend.workingDir" . }}/build/tmp
             - name: logs
-              mountPath: /opt/app-root/src/.npm/
+              mountPath: /home/node/
```

Comment:
Still thinking this needs to be configurable or automatic somehow. It was set to the previous value to get it to work at some point, and changing it away to make it work someplace else means it varies by distro/image used? Isn't changing this going to break it for someone else?